api/db/models.js

```diff
@@ -0,0 +1,8 @@
+import mongoose from 'mongoose';
+import { huntSchema } from 'schemas.js';
+
+const Hunts = mongoose.model('Hunts', huntSchema, 'Hunts');
```

Comment:
What's that 3rd parameter

Reply:
It's another parameter for the name I added it a while ago in one of my projects because it wouldn't work - I need to check the docs

Reply:
Just read through the docs and it only says that there are two params apart from where it talks about data replication:

```
 // Create a new mongoose model
  const personSchema = new mongoose.Schema({
    name: String
  });
  const Person = mongoose.model('Person', personSchema, 'Person');
```

This is the only place I could find where the three parameters are used.